pkg/kubectl/apps/kind_visitor_test.go

```diff
@@ -31,7 +31,7 @@ var _ = Describe("When KindVisitor accepts a GroupKind", func() {
 		visitor = &TestKindVisitor{map[string]int{}}
 	})
 
-	It("should Visit DaemonSet iff the Kind is a DaemonSet", func() {
```

Comment:
iff is generally short for "[if, and only if](https://en.wikipedia.org/wiki/If_and_only_if)"

Reply:
got it！

Reply:
@zjj2wry This is a typo, right?

For the root cause of this flake, we still got no clues.

Reply:
I don't think this is a typo. The usage of iff is correct in these tests.

Reply:
@dixudx this is not typo. it like Chinese four-character idiom. this can translate to ```当且仅当```
@spxtr cool 😊